qemu/tests/cfg/smbios.cfg

```diff
@@ -19,7 +19,7 @@
     # Please update these parameters based on your guest os system
     notset_output = "Not Specified"
     # RHEL uses different entries
-    Host_RHEL:
```

Comment:
The `Host_RHEL` variant is still used in QE's internal environment, so I am afraid that we have to keep it.

Reply:
Oh, this is where it came from. I knew it worked :smile:. Let's use both, then...